test/unit/remote-config/remote-config-api-client.spec.ts

```diff
@@ -664,6 +661,7 @@ describe('RemoteConfigApiClient', () => {
     });
   });
 
+  // eslint-disable-next-line @typescript-eslint/ban-types
```

Comment:
Plans to fix these two in the future? (either by specifying the function signature or allowing `Function`?)

Reply:
Yeah was going to keep it for a future PR but turns out it was an easy fix. Updated with a function signature. Thanks!